src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -1047,6 +1047,7 @@ public enum ADVF
         ADVF_PRIMEFIRST = 2,
     }
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Explicit)]
```

Comment:
If we are excluding this attribute, we should excluded `FieldOffsetAttribute` as well.
https://github.com/dotnet/corefx/blob/26b932e3c56702382d13ece010b300d3181d2891/eng/DefaultGenApiDocIds.txt#L31

Otherwise, we should keep this attribute.

Reply:
https://github.com/dotnet/corefx/issues/35602